tests/test_pipelines.py

```diff
@@ -1054,19 +1054,24 @@ def test_stable_diffusion_img2img_pipeline(self):
         output_image = ds[0]["image"].resize((768, 512))
 
         model_id = "CompVis/stable-diffusion-v1-4"
-        pipe = StableDiffusionImg2ImgPipeline.from_pretrained(model_id, use_auth_token=True)
+        pipe = StableDiffusionImg2ImgPipeline.from_pretrained(
+            model_id,
+            revision="fp16",  # fp16 to infer 768x512 images with 16GB of VRAM
```

Comment:
think we can test with smaller images here, no ?
`768x512 i` would too big for tests.

Reply:
Will update these in another PR, gotta draw a small owl or smth :joy:

Reply:
(also would like to rely less on the dataset indices for the reference images)